diff --git a/node_modules/@revenuecat/purchases-capacitor/.swiftpm/xcode/xcuserdata/gherold.xcuserdatad/xcschemes/xcschememanagement.plist b/node_modules/@revenuecat/purchases-capacitor/.swiftpm/xcode/xcuserdata/gherold.xcuserdatad/xcschemes/xcschememanagement.plist
new file mode 100644
index 0000000..66e5d85
--- /dev/null
+++ b/node_modules/@revenuecat/purchases-capacitor/.swiftpm/xcode/xcuserdata/gherold.xcuserdatad/xcschemes/xcschememanagement.plist
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>SchemeUserState</key>
+	<dict>
+		<key>RevenuecatPurchasesCapacitor.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>4</integer>
+		</dict>
+	</dict>
+</dict>
+</plist>
diff --git a/node_modules/@revenuecat/purchases-capacitor/ios/Sources/RevenuecatPurchasesCapacitor/PurchasesPlugin.swift b/node_modules/@revenuecat/purchases-capacitor/ios/Sources/RevenuecatPurchasesCapacitor/PurchasesPlugin.swift
index 81bf81e..e62c263 100644
--- a/node_modules/@revenuecat/purchases-capacitor/ios/Sources/RevenuecatPurchasesCapacitor/PurchasesPlugin.swift
+++ b/node_modules/@revenuecat/purchases-capacitor/ios/Sources/RevenuecatPurchasesCapacitor/PurchasesPlugin.swift
@@ -169,8 +169,9 @@ public class PurchasesPlugin: CAPPlugin, PurchasesDelegate, CAPBridgedPlugin {
 
     @objc func getCachedVirtualCurrencies(_ call: CAPPluginCall) {
         guard self.rejectIfPurchasesNotConfigured(call) else { return }
+        let cached = CommonFunctionality.getCachedVirtualCurrencies()
         call.resolve([
-            "cachedVirtualCurrencies": CommonFunctionality.getCachedVirtualCurrencies()
+            "cachedVirtualCurrencies": cached ?? NSNull()
         ])
     }
 
